modules/juce_audio_devices/native/juce_ios_Audio.cpp

```diff
@@ -552,9 +552,8 @@ struct iOSAudioIODevice::Pimpl      : public AudioPlayHead,
     bool setAudioPreprocessingEnabled (bool enable)
     {
         auto session = [AVAudioSession sharedInstance];
-
-        NSString* mode = (enable ? AVAudioSessionModeDefault
-                                 : AVAudioSessionModeMeasurement);
+                
+        NSString* mode = (enable ? AVAudioSessionModeVideoRecording : AVAudioSessionModeMeasurement);

```

Comment:
Why this change?

Reply:
`AVAudioSessionModeVideoRecording` supports changing the microphone position to the top one next to the camera (unlike `AVAudioSessionModeDefault`)

Reply:
`AVAudioSessionModeVideoRecording` has better recording quality than `AVAudioSessionModeDefault`

With `AVAudioSessionCategoryPlayAndRecord` we have 5 session modes:

Reply:
Just a note - this will also affect the case where the flag is disabled, but I think that's ok for now anyway.